foursight_core/app_utils.py

```diff
@@ -467,7 +914,212 @@ def view_foursight(self, environ, is_admin=False, domain="", context="/"):
+        bucket_names = {
+            "Environment Bucket Name:": self.environment.get_env_bucket_name(),
+            "Foursight Bucket Name:": get_foursight_bucket(envname=environ, stage=stage_name),
+            "Foursight Bucket Prefix:": get_foursight_bucket_prefix()
+        }
+        gac_name = get_identity_name()
+        gac_values = self.sorted_dict(obfuscate_dict(get_identity_secrets()))
+        environment_and_bucket_info = self.sorted_dict(obfuscate_dict(
+                                        self.environment.get_environment_and_bucket_info(environ, stage_name)))
+        declared_data = self.sorted_dict(EnvUtils.declared_data())
+        dcicutils_version = pkg_resources.get_distribution('dcicutils').version
+        foursight_core_version = pkg_resources.get_distribution('foursight-core').version
+        versions = {
+            # TODO/dmichaels/2022-08-04: Get the "Foursight-CGAP" value some other way,
+            # as the package might not be "Foursight-CGAP" (might be for example just "Foursight").
+            f"{self.html_main_title}:": self.get_app_version(),
+            "Foursight-Core:": foursight_core_version,
+            "DCIC-Utils:": dcicutils_version,
+            "Python:": platform.python_version()
+        }
+        resources = {
+            "Foursight Server:": socket.gethostname(),
+            "Portal Server:": self.get_portal_url(environ),
+            "ElasticSearch Server:": self.host,
+            "RDS Server:": os.environ["RDS_HOSTNAME"],
+            "SQS Server:": self.sqs.get_sqs_queue().url,
+        }
+        aws_credentials = self.get_obfuscated_credentials_info(environ)
+        aws_account_number = aws_credentials.get("AWS Account Number:")
+        os_environ = self.sorted_dict(obfuscate_dict(dict(os.environ)))
+        request_dict = request.to_dict()
+
+        html_resp.body = template.render(
+            request=request,
+            version=self.get_app_version(),
+            package=self.APP_PACKAGE_NAME,
+            env=environ,
+            env_short=short_env_name(environ),
+            env_full=full_env_name(environ),
+            domain=domain,
+            context=context,
+            environments=self.get_unique_annotated_environment_names(),
+            stage=stage_name,
+            is_admin=is_admin,
+            is_running_locally=self.is_running_locally(request_dict),
+            logged_in_as=self.get_logged_in_user_info(environ, request_dict),
+            user_record=self.user_record,
+            auth0_client_id=self.get_auth0_client_id(environ),
+            aws_credentials=aws_credentials,
+            aws_account_number=aws_account_number,
+            main_title=self.html_main_title,
+            favicon=self.get_favicon(),
+            load_time=self.get_load_time(),
+            init_load_time=self.init_load_time,
+            lambda_deployed_time=self.get_lambda_last_modified(),
+            running_checks='0',
+            queued_checks='0',
```

Comment:
Should these values be real? Should be able to extract them from SQS.

Reply:
I chose not to fetch and put them for the info for this (/info) page so as not to slow it down. In the future (if I continue my side React project) things like this will be pulled, asynchronously from React, individually from specific endpoints, for a more responsive UI.